scanpy/tests/external/test_nn_formatted.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+import scanpy as sc
+import scanpy.external as sce
+
+# Import neighbor testing functionality
+#from scanpy.tests.test_neighbors import get_neighbors
+from scanpy import Neighbors
+from scanpy.external.pp import nearest_neighbors_formatted
```

Comment:
mabye use "sce.pp" since you have defined "import scanpy.external as sce" on line 8

Reply:
for some reason doing that doesn't work